docs/health-api-specs/contracts/registries/individual.yml

```
@@ -1,597 +1,973 @@
-  offset:
-    name: offset
-    description: Pagination - offset from which records should be returned in response
-    in: query
-    type: integer
-    minimum: 0
-    required: true
+    lastChangedSince:
+      name: lastChangedSince
+      in: query
+      description: |
+        Epoch of the time since when the changes on the object should be picked up. 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)